doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -0,0 +1,98 @@
+"""
+============================================================
+Reconstruct dust-obstructed human cornea image with inpainting
+============================================================
+
+Optical Coherence Tomography (OCT) [1]_ is used to provide eye doctors with an
+image of the retina in the back of a patient's eye.
```

Comment:
Aww, I don't know how *retina* and *cornea* relate... It's always better to use terms from the title when you introduce a topic.